project/templates/app/mobi/registration/registration_form.html

```diff
@@ -32,13 +32,22 @@
 <input id="id_username" name="username" maxlength="30" type="text" class="input-wide" value="{{form.username.value|default:""}}">
 </label>
 
+<label for="id_username">Email
```

Comment:
@techaffinity I missed one. This email field is also required on the registration form in `base.html`. Please add it and submit in a new PR. I've created a new issue for it #19